granulate_utils/containers/cri.py

```diff
@@ -75,32 +77,37 @@ def list_containers(self, all_info: bool) -> List[Container]:
             with RuntimeServiceWrapper(path) as stub:
                 for container in stub.ListContainers(api_pb2.ListContainersRequest()).containers:
                     if all_info:
-                        # need verbose=True to get the info which contains the PID
-                        status = stub.ContainerStatus(
-                            api_pb2.ContainerStatusRequest(container_id=container.id, verbose=True)
-                        )
-                        pid: Optional[int] = json.loads(status.info.get("info", "{}")).get("pid")
+                        status_response = self._get_container_status(stub, container, verbose=all_info)
```

Comment:
```suggestion
                        status_response = self._get_container_status(stub, container, verbose=True)
```
and retain the comment please

Reply:
@michelhe you didn't do it, please retain the comment `# need verbose=True to get the info which contains the PID` and change to `verbose=True` for clarity.

Reply:
Did do, forgot to push sorry